docs/usage/exceptions.rst

```diff
@@ -116,3 +113,11 @@ As a result of the above structure, the exceptions raised by the ASGI Router its
 and ``405 Method Not Allowed`` are handled only by exception handlers defined on the app layer. Thus, if you want to affect
 these exceptions, you will need to pass the exception handlers for them to the Litestar constructor and cannot use other
 layers for this purpose.
+
+While it does not make much sense to have different functions with a top-level exception handling,
```

Comment:
I think if you're saying something doesn't make sense but can still be done, you also should explain:

- Why it doesn't make sense
- Why it can still be done
- If / When this should be used

Reply:
I removed this phrase. For this does make perfect sense: you customize the error handling for different routes.